source/administration/telemetry.rst

```diff
@@ -46,8 +46,12 @@ Mattermost error and diagnostic data is collected for the following purposes:
 .. note:: 
 Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. To opt out, you can disable the feature in **System Console > Environment > Logging** (or **System Console > General > Logging > Enable Error and Diagnostics Reporting** in versions prior to 5.12).
 
+Reporting Frequency
```

Comment:
Big +1 for being very specific about the frequency.

However, one suggestion: Mattermost has customers around the world and in many cases English is their second language. Few of the words and phrases used here have multiple meanings and may result in misunderstood terms.

One example: If I take `Every hour thereafter for the first 12 hours` and use [Google Translate](https://www.google.com/search?q=english+to+japanese&rlz=1C5CHFA_enCA768CA768&oq=english&aqs=chrome.1.69i57j69i59j0l4j69i59j0.1744j0j7&sourceid=chrome&ie=UTF-8) to translate it to Japanese, then back to English, I get `For the first 12 hours thereafter, every hour`, which is not quite the same information you intended to convey (but is what a non-native language speaking individual would get if they used a translator)

Thus, propose revisiting so that a non-English native speaker with basic computer skills can easily understand it, raising the quality of the section.